package-lock.json

```diff
@@ -5,641 +5,646 @@
   "requires": true,
```

Comment:
Was this what npm generated from a fresh install or something? Trying to understand what led to so many minor updates throughout the lockfile when there wasn't anything added to package.json

Reply:
I think that's why the linting rules changed slightly (I had to reformat some stuff to get the check to pass). In theory, minor version changes shouldn't break stuff, but this is a _lot_ at once.

Reply:
Unless there's some essential update in `package-lock.json`, I'm gonna revert that part of the PR (lock file changes + the lint changes it caused)